Alamofire.xcodeproj/project.pbxproj

```diff
@@ -1683,11 +1678,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				OTHER_LDFLAGS = (
-					"-framework",
-					CFNetwork,
```

Comment:
Need the linking here too.

Reply:
Hello @jshier 
Please check this comment https://github.com/Alamofire/Alamofire/pull/3112#discussion_r392713193.